charts/seed-bootstrap/charts/loki/templates/statefulset.yaml

```
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: loki
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ toYaml .Values.labels | indent 4 }}
+  annotations:
+    {{- toYaml .Values.annotations | nindent 4 }}
```

I guess you could use the same `toYaml ...` 
```suggestion
 {{ toYaml .Values. | annotationsindent 4 }}
```